syntax = "proto3";
package Register; 

message RegisterRequest{
    string username = 1;
    string password = 2;
    bool sex = 3;
    string telphone = 4;
    string address = 5;
}

message RegisterResponse{
    int32 status = 1;
}

service RegisterService{
    rpc Register(RegisterRequest) returns (RegisterResponse) {}
}